src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -489,22 +681,22 @@ void func_80996F98(DoorShutter* this, PlayState* play) {
     }
 }
 
-void func_80997004(DoorShutter* this, PlayState* play) {
-    if (DECR(this->unk_16E) == 0 && play->roomCtx.status == 0 && func_80996D14(this, play) != 0) {
+void DoorShutter_Opening(DoorShutter* this, PlayState* play) {
+    if (DECR(this->lockTimer) == 0 && play->roomCtx.status == 0 && DoorShutter_UpdateOpening(this, play)) {
         if (((this->doorType == SHUTTER_BOSS) ? 20.0f : 50.0f) < this->dyna.actor.xzDistToPlayer) {
```

Comment:
Open and close is probably fine instead of opening and closing?

Reply:
I'm not done with DoorShutter, there are probably action funcs that would like to use "open" as a "idle open" action but idk

In the meantime I can unname these actions if you want

Reply:
its fine to leave it named. can wait to see how the rest of the docs go and reevaluate this later. I just personally like the actions to be simple verbs instead of -ing